packages/react-email/package.json

```diff
@@ -53,7 +53,7 @@
     "next": "^15.3.2",
     "react": "19.0.0",
     "react-dom": "19.0.0",
-    "tsup": "8.4.0",
+    "tsdown": "^0.15.1",
```

Comment:
can we use pinned dependencies here?

Reply:
also, I'm wondering why we're defining `tsdown` as a `devDependency` here and not for other packages